easycv/apis/export.py

```diff
@@ -160,31 +181,87 @@ def _export_yolox(model, cfg, filename):
 
     if hasattr(cfg, 'export') and (getattr(cfg.export, 'use_jit', False) or
                                    getattr(cfg.export, 'export_blade', False)):
-        device = 'cuda' if torch.cuda.is_available() else 'cpu'
 
+        # only when we use jit or blade, we need to reparameterize_models before export
+        model = reparameterize_models(model)
+        device = 'cuda' if torch.cuda.is_available() else 'cpu'
         model = copy.deepcopy(model)
-        model.eval()
-        model.to(device)
 
         end2end = cfg.export.get('end2end', False)
         if LooseVersion(torch.__version__) < LooseVersion('1.7.0') and end2end:
             raise ValueError('`end2end` only support torch1.7.0 and later!')
 
         batch_size = cfg.export.get('batch_size', 1)
+        static_opt = cfg.export.get('static_opt', True)
+        use_blade = getattr(cfg.export, 'export_blade', False)
+        use_trt_efficientnms = cfg.export.get('use_trt_efficientnms', False)
+
+        # assert image scale and assgin input
         img_scale = cfg.get('img_scale', (640, 640))
+
         assert (
             len(img_scale) == 2
         ), 'Export YoloX predictor config contains img_scale must be (int, int) tuple!'
 
         input = 255 * torch.rand((batch_size, 3) + img_scale)
+        print(input.dtype)
```

Comment:
is this log necessary?   if yes, add more text to describe it

Reply:
we have deleted it.